packages/ove-app-svg/src/client/view/svg.js

```diff
@@ -0,0 +1,29 @@
+initView = function () {
+    window.ove.context.isInitialized = false;
+    log.debug('Application is initialized:', window.ove.context.isInitialized);
+};
+
+getCSS = function () {
+    const l = window.ove.layout;
+    // The webpage is plotted across the entire section and then
+    // moved into place based on the client's coordinates.
+    const css = {
+        transform: 'translate(-' + l.x + 'px,-' + l.y + 'px)',
+        width: l.section.w + 'px',
+        height: l.section.h + 'px'
+    };
+    log.debug('Resizing viewer with height:', css.height, ', width:', css.width);
+    log.debug('Performing CSS transform on viewer', css.transform);
+    return css;
+};
+
+beginInitialization = function () {
+    log.debug('Starting viewer initialization');
+    // Nothing to run post-init.
```

Comment:
What is this comment supposed to mean?

- Why would there be something to run post-init?
- It's also confusing because there *is* something to run after ``OVE.Utils.initView`` (the anonymous function passed as a callback), just not after ``beginInitialization``.

Reply:
This comment does not make sense. The function that you find which runs after `OVE.Utils.initView` was added later on, and it is my mistake to have kept this log as-is. This will be deleted in the upcoming commit.